src/services/userService.js

```diff
@@ -239,7 +272,8 @@ function createToken(user) {
         // address: user.address,
         phone: user.phone,
         isDeleted: user.isDeleted,
-        accessToken: jwt.sign(payload, secret),
+        isApproved: user.isApproved,
+        accessToken: jwt.sign(payload, secret, { expiresIn: '24h' }),
```

Comment:
Why 24h ? Make it like 5-10 days

Reply:
fixed with seconds. In documentation is said to use seconds instead of days.